docs/talk2biomodels/cases/Case_2.md

```diff
@@ -0,0 +1,148 @@
+  <li><strong>beta:</strong> Transmission probability per contact.</li>
+  <li><strong>q:</strong> Quarantine rate.</li>
+  <li><strong>sigma:</strong> Rate of progression from exposed to infected.</li>
+  <li><strong>lambda:</strong> Rate of recovery.</li>
+  <li><strong>rho:</strong> Rate of hospitalization.</li>
+  <li><strong>delta_I:</strong> Rate of death for infected individuals.</li>
+  <li><strong>delta_q:</strong> Rate of death for quarantined individuals.</li>
+  <li><strong>gamma_I:</strong> Recovery rate for infected individuals.</li>
+  <li><strong>gamma_A:</strong> Recovery rate for asymptomatic individuals.</li>
+  <li><strong>gamma_H:</strong> Recovery rate for hospitalized individuals.</li>
+  <li><strong>alpha:</strong> Rate of symptomatic infection.</li>
+  <li><strong>theta:</strong> Effectiveness of interventions.</li>
+  <li><strong>c_lockdown:</strong> Contact rate during lockdown.</li>
+  <li><strong>q_lockdown:</strong> Quarantine rate during lockdown.</li>
+  <li><strong>Infected:</strong> Number of infected individuals.</li>
+</ul>
+
+<h3>Compartments:</h3>
+<ul>
+  <li><strong>Wuhan:</strong> The geographical compartment where the model is focused.</li>
+</ul>
+
+<h3>Units:</h3>
+<ul>
+  <li><strong>Time Unit:</strong> Days (d)</li>
+  <li><strong>Quantity Unit:</strong> Number of individuals (#)</li>
+  <li><strong>Length Unit:</strong> Meters (m)</li>
+  <li><strong>Area Unit:</strong> Square meters (m²)</li>
+  <li><strong>Volume Unit:</strong> Milliliters (ml)</li>
+</ul>
+
+<p>These components work together to simulate the dynamics of the COVID-19 outbreak and the impact of public health interventions.</p>
+</details>
+
+
+<hr>
+
+### Compute infected cases over time
+**Q3** Simulate the model and plot infected cases over time <br>
+
+```
+Simulate the model for 50 days with an interval of 50. Plot infected cases over time. Call this model_default.
+```
+
+<details>
+<summary>Answer</summary>
+<img src="../figures/C2_q3.png" width="600"/>
+</details>
+
+
+**Q4** Set the quarantine rate (`q_lockdown`) 20 times the initial value (=3.7774 e-06) and simulate the model (reproduce the red line in figure 3C from the paper). <br>
+
+
+```
+Simulate the model again, but set the parameter 'q_lockdown' to 0.0000037774. Call this model_high_lockdown.
+```
+
+```
+Plot the infected cases over time.
+```
+
+<details>
+<summary>Answer</summary>
+<img src="../figures/C2_q4.png" width="600"/>
+</details>
+
+
+**Q5:** Compare infected cases for both simulations. <br>
+
+```
+What is the value of infected cases at the end of the simulation in both experiments?
+```
+
+<details>
+<summary>Answer</summary>
+
+The values of infected cases at the end of the simulations are as follows:
+<ul>
+  <li>For the experiment model_default, the value is approximately <strong>104,338.52</strong></li>
+  <li>For the experiment model_high_lockdown, the value is approximately <strong>7,275.49</strong></li>
+</ul>
+</details>
+
```

Comment:
Unless there is a reason for them, you can omit the trailing new lines at at the end of the file.